package.json

```diff
@@ -29,12 +29,12 @@
     "state channels"
   ],
   "devDependencies": {
-    "@types/prettier": "1.18.2",
+    "@types/prettier": "1.19.0",
```

Comment:
Does it make sense to put prettier (and other common devDependencies) in the root `package.json`?

Reply:
Here's a related eslint rule, which would let us enforce that:
- anything that is not listed as a `dependency` in a package, cannot be imported by that package

We would need to configure that rule to only apply to the source files of the project, since we would probably want use dev dependencies eg. in test files.

Reply:
Hmm, I am not sure. We should discuss this.
I am going to leave this out of this PR.
Can you please open a ticket with these concerns?